android-lib/src/androidTest/assets/kinvey.properties

```diff
@@ -1,6 +1,6 @@
 # Required settings
-app.key=<YOUR_APP_KEY_HERE>
-app.secret=<YOUR_APP_SECRET_HERE>
+app.key=kid_BkssrV2eW
```

Comment:
@yuliya-guseva do these keys need to be included here, or is it an accident? We prefer not to have any keys in public, so I suggest removing these from here and adding them as environment variables if necessary.